examples/stm32h7/src/bin/dac.rs

```diff
@@ -54,3 +52,16 @@ fn to_sine_wave(v: u8) -> u8 {
         (r.sin() * 128.0 + 127.0) as u8
     }
 }
+
+fn config() -> Config {
+    let mut config = Config::default();
+    config.rcc = rcc_config();
+    config
+}
+
+fn rcc_config() -> rcc::Config {
+    let mut config = rcc::Config::default();
+    config.sys_ck = Some(400.mhz().into());
+    config.pll1.q_ck = Some(100.mhz().into());
+    config
+}
```

Comment:
Duplicated for each h7 example. Could be moved to `example_common`

Reply:
I would make it a single method too. `config.rcc.sys_ck = xx`